packages/renderers-js/e2e/anchor/idl.json

```diff
@@ -366,6 +366,15 @@
     }
   ],
   "types": [
+    {
+      "name": "BurnArgs",
+      "type": {
+        "kind": "enum",
+        "variants": [
+          { "name": "V1", "fields": [{ "name": "amount", "type": "u64" }] }
+        ]
+      }
+    },
```

Comment:
This is from the `mpl-token-metadata` program and fails the test on `main` due to this bug